opencog/atomese/interpreter/logical_interpreter.h

```diff
@@ -147,5 +147,22 @@ bool logical_compare(const LinkValuePtr& p1, const LinkValuePtr& p2){
 
 	return std::equal(p1_value.begin(), p1_value.end(), p2_value.begin(), comparator);
 }
+
+/**
+ * compute logical_not.
+ * it expects the values of the LinkValues to be TRUE_LINK and FALSE_LINK
+ * @param LinkValuePtr&     p1
+ *
+ * @return                 LinkValue pointer containing the logical_not.
+ */
+LinkValuePtr logical_not(const LinkValuePtr& p){
```

Comment:
Very minor, but it seems you need either a space or a new line between `)` and `{`.